src/content/documentation/develop/ux-guidelines-for-mobile-extensions.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: sidebar
+title: Differences between desktop and Android extensions
+permalink: /documentation/develop/ux-guidelines-for-mobile-extensions/
+topic: Develop
+tags: [add-ons, guide, mobile, webextensions, ux, user-experience]
+contributors:
+  [
+    caitmuenster
+  ]
+last_updated_by: caitmuenster
+date: 2020-10-19
+---
+
+<!-- Page Hero Banner -->
+
+{% capture page_hero_banner_content %}
+
+# User Experience Guidelines for Mobile Extensions
+
+TEST TEST TEST
+
+{% endcapture %}
+{% include modules/page-hero.liquid
+    content: page_hero_banner_content
+%}
+
+<!-- END: Page Hero Banner -->
+
+<!-- Content with Table of Contents Module -->
+
+{% capture content_with_toc %}
+
+# Introduction
+
+::: note alert
+In August 2020, Mozilla launched a new, reimagined Firefox for Android experience (codenamed "Fenix"). The  browser for Android has been rebuilt from the ground up using [GeckoView](https://mozilla.github.io/geckoview/), Mozilla's mobile browsing engine.
```

Comment:
Maybe want to link https://blog.mozilla.org/blog/2020/08/25/introducing-a-new-firefox-for-android-experience/ ?